src-java/ping-topology/ping-storm-topology/src/main/java/org/openkilda/wfm/topology/ping/bolt/FlowFetcher.java

```diff
@@ -251,6 +261,54 @@ private void handleOnDemandYFlowRequest(Tuple input) throws PipelineException {
         }
     }
 
+    /**
+     * Handle on demand ha flow ping request.
```

Comment:
multiple people use lowercase for ha. It's ok in variable or class names, but in comments or error messages I'd suggest to stick with the standard English: use upper case for abbreviations. I'd suggest to avoid using hyphen as some people do. My view: 
* don't: ha-flow, ha flow
* do: HA flow or HA Flow (a capital F to indicate that it is a model or a Java type, not just a flow in general)